content/applications/finance/accounting/bank/reconciliation.rst

```diff
@@ -2,94 +2,146 @@
+#. If the resulting entry isn't fully balanced, balance it by adding another existing counterpart
+   entry or writing it off with a :ref:`manual operation <reconciliation/manual-operations>`.
+#. Click the :guilabel:`Validate` button to confirm the reconciliation and move to the next
+   transaction.
 
-**Import** or **Create** the bank statements. Please refer to the
-documents from the Bank Feeds section.
+.. tip::
+   If you aren't sure how to reconcile a particular transaction and would like to deal with it
+   later, use the :guilabel:`To Check` button instead. All transactions marked as :guilabel:`To
+   Check` can be displayed using the :guilabel:`To Check` filter.
 
-.. image:: reconciliation/use06.png
+.. note::
+   Bank transactions are posted on the **suspense account** until reconciliation. At this point,
+   reconciliation modifies the transaction journal entry by replacing the bank suspense account with
+   the corresponding receivable, payable, or outstanding account.
 
-On the dashboard, click on **Reconcile # Items**
+.. _reconciliation/existing-entries:
 
-.. image:: reconciliation/use07.png
+Match existing entries
+----------------------
 
-If everything was right (correct partner name, right amount) odoo will
-do the reconciliations **automatically**.
+The :guilabel:`Match Existing Entries` tab reflects the journal items from your business records
+(such as customer invoices, vendor bills, and payments). The entry order is based on
+:ref:`reconciliation models <reconciliation_models_suggestion>`, with suggested entries appearing
+first.
 
-.. image:: reconciliation/use08.png
+.. tip::
+   The search bar within the :guilabel:`Match Existing Entries` tab allows you to search for
+   specific journal items.
 
-If some issues are found, you will need to take **manual actions**.
+.. _reconciliation/batch-payments:
 
-For example, if the partner is missing from your bank statement, just
-fill it in :
+Batch payments
+--------------
 
-.. image:: reconciliation/use09.png
+Use the :guilabel:`Batch Payments` tab to find batch payments for customers and vendors. Similarly
+to the :guilabel:`Match Existing Entries` tab, the :guilabel:`Batch Payments` tab has a search bar
+that allows you to search for specific batch payments.
 
-If the payment is done with a down payment, just check if it is all
-right and validate all related payments :
+.. _reconciliation/manual-operations:
 
-.. image:: reconciliation/use10.png
+Manual operations
+-----------------
 
-.. seealso::
-   :doc:`bank_synchronization`
+If there isn't an existing entry to match the selected transaction, you may instead wish to
+reconcile the transaction manually by choosing the correct account and amount. Then, complete any
+of the relevant optional fields.
+
+.. note::
+   If you want to set a record as entirely paid, even if only a partial payment has been made for
+   this transaction, click :guilabel:`fully paid`.
```

Comment:
More explanation is required here. What happens with the missing amount?
--> open balance on the account receivable

and even, what does it mean to "mark as paid"?. The fact that a record is marked as paid is explained in the payment section, maybe add a link.